python/cuml/test/utils.py

```diff
@@ -30,6 +32,9 @@
 import cuml
 import pytest
 
+# max_gpu_memory: Capacity of the GPU memory in GB
+max_gpu_memory = 0
```

Comment:
I think using pytest_configure might be a better choice as opposed to the global variable https://docs.pytest.org/en/6.2.x/reference.html#pytest.hookspec.pytest_configure